### ISSUE TITLE
Commercial Skip (Hardware) 

### ISSUE THREAD

**Issue by CollabSensei** [cedarrapidsboy/pms-postprocessing#7]
The current release version of Comskip that is distributed with Plex (public version), does not include hardware/GPU support. The true donator version of Comskip supports gpu analysis.

Here is what I did:
1. Clone Comskip Linux repo - https://github.com/erikkaashoek/Comskip
2. Compile the script and copy the file to the directory where Plex Commercial Skipper is located.  named it Plex Commercial Skipper 2
3. Modified the script to launch 'Plex Commercial Skipper2' and pass the following arguments: --hwassist --cuvid
4. Add "hardware_decode=1", to the comskip.ini.

I need to move the comskip.ini and 'Plex Commercial Skipper 2' into a different directory, so it is not overwritten by Plex upgrades.  I haven't had a chance to do that. When doing the commercial analysis in software, I was getting around 90fps. In Hardware (P2000), I get around 400-600fps, and cpu utilization isn't pegged at 100%.

**Comment by cedarrapidsboy** [comment]
I'm glad you got that working. I played around with it and finally got comskip to build. Unfortunately, in my tests I was not able to get HW decoding to kick-in. The option was available (--hwassist --cuvid) and recognized. The INI was updated (hardware_decode=1). And, "nvidia-smi" reported that everything was ready to go. But, during execution, no decoding was done on the GPU. Perhaps I do not have a cuvid-enabled ffmpeg build.

Thanks for your instructions. I hope PMS adds --cuvid into their OOTB comskip build. Including a custom build of it now would require more dependencies than I would want (ffmpeg, libargtable).
